tests/test_models.py

```diff
@@ -213,9 +212,216 @@ def test_wishlist_id_is_increment(self):
         wishlist.create()
-        expected = wishlist.wishlist_id + 1
+        expected = wishlist.id + 1
 
         for wishlist in WishlistFactory.create_batch(5):
             wishlist.create()
-            self.assertEqual(wishlist.wishlist_id, expected)
+            self.assertEqual(wishlist.id, expected)
             expected += 1
+
+
+#####################
+# WishistItem Tests
+#####################
+class TestWishlistItem(unittest.TestCase):
+    """Test cases for WishlistItem Model"""
+
+    serialized = {}
+
+    @classmethod
+    def tearDownClass(cls):
+        """This runs once after the entire test suite"""
+        db.session.query(WishlistItem).delete()
+        db.session.commit()
+
+    def setUp(self):
+        """This runs before each test"""
+        db.session.query(WishlistItem).delete()
+        db.session.commit()
+
+        self.serialized = {
+            "id": 1,
+            "wishlist_id": 2,
+            "product_id": 3,
+            "product_name": "DevOps for Dummies",
+            "product_price": 29.99,
+            "quantity": 2,
+            "created_date": datetime.strptime(
+                "2023-10-12 00:00:00", "%Y-%m-%d %H:%M:%S"
+            ),
+        }
+
+    def tearDown(self):
+        """This runs after each test"""
+        db.session.remove()
+
+    def test_wishlist_item_init_clears_id(self):
+        """It should ensure id is set to None upon initialization"""
+        item = WishlistItem()
+        item.__init__()
+        self.assertIsNone(item.id)
+
+    def test_wishlist_item_no_arg_initializer(self):
+        """It should create an instance using the no arg initializer"""
+        item = WishlistItem()
+        self.assertIsNotNone(item)
+
+    # Note: This is kind of a silly test.  I had problems trying to implement a __init__ function that would have set all these values in a single line.
+    def test_wishlist_after_populating_properties(self):
+        """It should create an instance using the constructor with arguments"""
+        now = datetime.now()
+        values = {
+            "id": 1,
+            "wishlist_id": 2,
+            "product_id": 3,
+            "product_name": "The Big Lebowski",
+            "product_price": 9.99,
+            "created_date": now,
+        }
+        item = WishlistItem()
+        item.id = values["id"]
+        item.wishlist_id = values["wishlist_id"]
+        item.product_id = values["product_id"]
+        item.product_name = values["product_name"]
+        item.product_price = values["product_price"]
+        item.created_date = values["created_date"]
+
+        # self.assertEqual(item.id, values.id)
+        self.assertEqual(item.wishlist_id, values["wishlist_id"])
+        self.assertEqual(item.product_id, values["product_id"])
+        self.assertEqual(item.product_name, values["product_name"])
+        self.assertEqual(item.product_price, values["product_price"])
+        self.assertEqual(item.created_date, values["created_date"])
```

Comment:
I think it's fine for now. 
I'm not sure if anyone else has suggestions for alternatives, but it can be added as a story to make it leaner if we have the time.